### PR TITLE
Add a GitHub workflow for Build/Pushing the Dev container images

### PR DESCRIPTION
**Pull request by SpencerLN**
Create a workflow to automate the process of building and pushing the container images for the Dev branch.

### CODE REVIEW

**Review by ajrios33 — APPROVED** [review]
LGTM

**Comment by SpencerLN** [comment]
Going to wait for a review from @ismisepaul as well since he is more familiar with this application.

**Review by ismisepaul — APPROVED** [review]
LGTM

**Pull request merged**
